templates/prescription/general_lookup.html

```diff
@@ -7,12 +7,14 @@
  * @copyright Copyright (c) 2017 Brady Miller <brady.g.miller@gmail.com>
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  *}
+<!DOCTYPE html>
 <html>
 <head>
 
 {headerShow}
-
+<link rel="stylesheet" href="{$GLOBALS.assets_static_relative}/bootstrap-3-3-4/dist/css/bootstrap.min.css">
 <link rel="stylesheet" href="{$GLOBALS.css_header}" type="text/css">
```

Comment:
In smarty, the following would does the standard header (would replace lines 14, 15, 16):
``````
{headerTemplate}
``````
Also for future can also have it bring in additional assets via below example:
``````
{headerTemplate assets='bootstrap-sidebar|common'}
``````

Reply:
Quick reference on this smarty function is here:
https://github.com/openemr/openemr/blob/master/library/smarty/plugins/function.headerTemplate.php